client/blocks/login/index.jsx

```diff
@@ -58,15 +59,20 @@ class Login extends Component {
 			rememberMe,
 		} = this.state;
 
-		if ( twoStepNonce && twoFactorAuthType === 'code' ) {
+		if ( twoStepNonce && ( twoFactorAuthType === 'code' || twoFactorAuthType === 'sms' ) ) {
```

Comment:
Previously, `code` was used because it describes the auth type of SMS codes, backup codes, and Authenticator app codes. If we're going to use more granular auth types, I think we should change code to `authenticator` to match the types returned by the server and prevent confusion (since `sms` uses "codes" too).